Sources/Hedera/ChunkedTransaction.swift

```diff
@@ -102,6 +102,14 @@ public class ChunkedTransaction: Transaction {
         return super.addSignatureSigner(signer)
     }
 
+    internal final func getAllSignatures() throws -> [[AccountId: [PublicKey: Data]]] {
+        let sources = try self.makeSources()
+
+        self.sources = sources
+
+        return try Array(sources.chunks.map { try Transaction.getSignaturesAtOffset(chunk: $0) })
```

Comment:
```suggestion
        return try sources.chunks.map { try Transaction.getSignaturesAtOffset(chunk: $0) }
```
If it compiles, if it doesn't compile then leave it as-is.